src/main/kotlin/org/opensearch/commons/alerting/model/Monitor.kt

```diff
@@ -44,7 +44,8 @@ data class Monitor(
     val dataSources: DataSources = DataSources(),
     val deleteQueryIndexInEveryRun: Boolean? = false,
     val shouldCreateSingleAlertForFindings: Boolean? = false,
-    val owner: String? = "alerting"
+    val owner: String? = "alerting",
+    val fanoutEnabled: Boolean? = true
```

Comment:
This attribute would be specific to doc level monitors, right? Should it instead be in the doc level monitor input then?
https://github.com/opensearch-project/common-utils/blob/main/src/main/kotlin/org/opensearch/commons/alerting/model/DocLevelMonitorInput.kt

Reply:
addressed